cypress/helpers/dom.helper.ts

```diff
@@ -47,9 +37,7 @@ export const DOM = {
       get elements() {
         return {
           get cy() {
-            return DOM.list.cy.within(() => {
-              cy.get(`*[data-list-item]`);
-            });
```

Comment:
This is a bit of a weird one but the tests were failing because of that (painful to debug :sweat:).

Here's the corresponding thread: https://github.com/cypress-io/cypress/issues/8873